code/numerical_analysis/integral/integral_rectangle.cpp

```diff
@@ -0,0 +1,34 @@
+#include <iostream>
+#include <functional>
+
+//Part of Cosmos by OpenGenus Foundation
+double f(double x)
```

Comment:
What does this function do? Why is this even necessary in your code?

Reply:
f is just example for examination to work of integral. Better delete?

Reply:
f is symbol of mathematics function.

Reply:
Oh, I see, but it seems somewhat redundant, since this is code, anyways.